profile/Makefile

```diff
@@ -177,6 +184,9 @@ ifeq ($(OPTIMIZE),yes)
   ifeq ($(COMPILER),intel)
     CFLAGS += -O3 -xhost -fast -ansi-alias -no-prec-div -ipo
```

Comment:
-fast gives me a link error using icpc. It is default linking static lib. Better delete it.

Reply:
ok done. I read about that when deleting that flag for the MPI wrapped Intel compiler. It wasn't an issue on my side to statically link the stdlibc++ but it was for the mpi ones